users/models.py

```diff
@@ -362,11 +361,11 @@ class BankTransaction(models.Model):
     )
 
     def __str__(self):
-        return 'Bank transaction for ' + (self.user.email if self.user else 'unknown user') \
-            + ' from ' + self.sender \
-            + ' ' + str(self.amount) + '€, reference ' + str(self.reference_number) \
-            + (', message ' + self.message if self.message else '') \
-            + ' at ' + str(self.date)
+        return f'Bank transaction for {self.user.email if self.user else "unknown user"}' \
```

Comment:
fyi: {self.user.email or 'unknown user'} should work also (bit shorter and nicer to read :)